### DISCUSSION TITLE
Does kitty support "Enquiry character"

### DISCUSSION THREAD

**Discussion by nutilius** [kovidgoyal/kitty#7467] [Q&A]
Does kitty support "Enquiry character" (https://en.wikipedia.org/wiki/Enquiry_character)?
iTerm returns "iTerm" string, PuTTY has configuration option (default string "PuTTY"). 
It gives the chance to get additional info from terminal, even if TERM variable is remapped to another value (like   xterm-kitty to xterm).

**Comment by kovidgoyal** [comment]
If you want to query a terminal for its capabilities including name, use
the XTGETTCAP escape code.
